src/persistence/secondary/mongo-secondary-entity-manager.ts

```diff
@@ -0,0 +1,77 @@
+import { Entity } from '@antjs/ant-js/build/ant';
+import { Collection, MongoClient } from 'mongodb';
+import { MongoBuildArgs } from '../../model/mongo-build-args';
+import { MongoModel } from '../../model/mongo-model';
+import { SecondaryEntityManager } from './secondary-entity-manager';
+
+export class MongoSecondaryEntityManager<TEntity extends Entity> implements SecondaryEntityManager<TEntity> {
+  public get model(): MongoModel<TEntity> {
+    return this._model;
+  }
+
+  protected _model: MongoModel<TEntity>;
+  private _collection: Promise<Collection>;
+  private _client: Promise<MongoClient>;
+
+  constructor(model: MongoModel<TEntity>, args: MongoBuildArgs) {
+    this._model = model;
+    this._client = MongoClient.connect(args.url);
+    this._collection = this._client.then((client) => client.db(args.dbName).collection(this._model.collectionName));
+  }
+
+  public async delete(id: string | number): Promise<any> {
+    return (await this._collection).findOneAndDelete({ id: id });
```

Comment:
findOneAndDelete has an extra cost returning the removed document. Since we don't need the document, would it be a good idea to use the  "deleteOne" operation instead?

Reply:
Yeah, that´s a great idea. I will remove it